R/merge_sce_list.R

```diff
@@ -237,10 +237,30 @@ merge_sce_list <- function(
           preserve_rowdata_cols = unique(altexp_rowdata_cols)
         )
     }
-  }
 
-  # Create the merged SCE from the processed list
-  merged_sce <- do.call(combineCols, unname(sce_list))
+    # if using alt exp, need to merge and then convert to dgCMatrix
```

Comment:
For modularity, I think it would be better to have a separate block `if` block for the actual merging step. The way it is now, the merging of non-altexp is nested under a header of "Handle altExps", which seems odd. For my own preference of trying to have the short option in an `if...else` block first, I might reverse the order; testing `if(!include_altxp)`.  But that seems less important than the modularity part.